recipe/meta.yaml

```diff
@@ -10,31 +10,35 @@ source:
   sha256: a3a38e67fdfcda94df51c805343150016097f284771156f76839a9c3a24c90a9
 
 build:
-  # Skipping build on linux, because a shorter prefix-length is required which is only possible by
-  # modifying the build string. Skipping build on py2k because there are some issues on AppVeyor
-  skip: True  # [linux or py2k]
+  # Skipping build on py2k because there are some issues on AppVeyor
+  skip: True  # [py2k]
   number: 0
 
 requirements:
   build:
-    - {{ compiler('c') }}  # [not win]
-    - clangdev  # [linux]
-    - pkg-config  # [not win]
+    - {{ compiler('c') }}   # [not win]
+    - clangdev              # [linux]
+    - pkg-config            # [not win]
   host:
     - glib  # [not win]
     - mono  # [not win]
     - pip
     - pycparser
     - python
   run:
-    - gettext  # [not win]
-    - libiconv  # [not win]
-    - mono  # [not win]
+    - {{ compiler('c') }}  # [linux]
+    - gettext              # [not win]
+    - libiconv             # [not win]
+    - mono                 # [not win]
     - python
 
 test:
   imports:
-    - clr
+    - clr  # [not linux]
+  commands:
+    - PY_LIBDIR=$($PYTHON -c 'import sysconfig; print(sysconfig.get_config_var("LIBDIR"))')  # [linux]
+    - export LD_LIBRARY_PATH=$PY_LIBDIR:$PREFIX/x86_64-conda_cos6-linux-gnu/sysroot/usr/lib:$LD_LIBRARY_PATH  # [linux]
```

Comment:
I don't think we should merge this PR even if its works. This is super wired. At worst we should move this to an env activation hook.

Reply:
Me too, I just wanted to try if we get it working.

Reply:
Uha, there are a few more in conda-forge that already had this problem:

https://github.com/conda-forge/r-pca3d-feedstock/commit/7e9fb5d0c603eb150fcd2d56a8f36952c1ca313e

Reply:
Interestingly this package works normally so maybe this is really a conda-forge build/test thing.

Can you mirror the commit above and test this?

Reply:
What should I mirror?
* The test command with setting the environment variable? It’s quite similar to mine. 
* The CDT packages? Those shouldn‘t help us at runtime, or?

We could remove the compiler from the runtime requirements and use the compiler dir from the build environment

Reply:
The test section and the build.sh script.

Reply:
Seems the variable is not needed in the build-script. I removed the runtime dependency of the compiler and simplified the testing commands.